app/controllers/jobs_controller.rb

```diff
@@ -1,6 +1,7 @@
 class JobsController < ApplicationController
   def index
     @company = Company.find(params[:company_id])
+    @locations = Location.all
     @jobs = @company.jobs
```

Comment:
Hmm, why would you want to send all locations from your database back with the company?